apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -67,6 +67,9 @@ class OrderBlocksAnswerData(TypedDict):
 FIRST_WRONG_TYPES = frozenset(
     [FeedbackType.FIRST_WRONG, FeedbackType.FIRST_WRONG_VERBOSE]
 )
+RANKING_DAG_ORDERED_GRADING_TYPE = frozenset(
+    [GradingMethodType.RANKING, GradingMethodType.DAG, GradingMethodType.ORDERED]
+)
```

Comment:
I don't see why define this if it is only used in one place. But it seems there should be multiple places it gets used, once you fix the `test` function in this file.

You should add an example of to the `orderBlocks.html` question in the `testCourse`. So that the automated tests ensure that the new feature you are adding is working correctly.

Reply:
This was used in two places (line 657 and 801), as suggested by @\eliotwrobson. It will probably be handy too in the `test` functions like you said, so I'm thinking we should keep it.

I will look at changing the `orderBlocks.html` question.